bberg/src/circuit_builder.rs

```diff
@@ -254,6 +258,60 @@ class {name}CircuitBuilder {{
             &circuit_hpp,
         );
     }
+
+    fn create_circuit_builder_cpp(&mut self, name: &str, all_cols_with_shifts: &[String]) {
```

Comment:
@fcarreiro If this outputs also the "shift columns", I would rather make the inclusion of such columns optional. We will rarely need them. It will be rare that a bug emerges from a malformed shift column.

Reply:
done!